src/bootstrap/dist.rs

```diff
@@ -402,6 +402,10 @@ impl Step for Rustc {
             if builder.config.lld_enabled {
                 let exe = exe("rust-lld", compiler.host);
                 builder.copy(&src_dir.join(&exe), &dst_dir.join(&exe));
+                // for `-Z gcc-ld=lld`
+                let gcc_lld_dir = dst_dir.join("gcc_ld");
```

Comment:
```suggestion
                let gcc_lld_dir = dst_dir.join("gcc-ld");
```
Nit: for style compatibility with naming in other places.